src/components/uv-scroll.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Animate the UV offset of a mesh's material
+ * @component uv-scroll
+ */
+AFRAME.registerComponent("uv-scroll", {
+  schema: {
+    speed: { type: "vec2", default: { x: 0, y: 0 } }
```

Comment:
nit: May call this `velocity` because it has direction and magnitude

Reply:
Was matching similar properties we already have on particle systems, and also didn't want it to incorrectly be associated with physics.